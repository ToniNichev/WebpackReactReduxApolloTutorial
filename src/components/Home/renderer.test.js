import React from 'react';
import { shallow, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Component from './renderer';

describe('Testing Home component', () => {
    it('renders as expected!!!', () => {
      const wrapper = shallow(<Component styles={{}} title="MyTest" />);

     expect(toJson(wrapper)).toMatchSnapshot();
     expect(wrapper.contains(<div>MyTest</div>)).toBe(true);
   });
});