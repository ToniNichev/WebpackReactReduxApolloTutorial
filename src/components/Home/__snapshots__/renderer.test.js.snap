// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Home component renders as expected!!! 1`] = `
<div>
  <div>
    MyTest
  </div>
</div>
`;
